_opensearch/search-backpressure.md

```diff
@@ -0,0 +1,217 @@
+              "current_max": "731.3ms",
+              "current_max_millis": 731,
+              "current_avg": "303.6ms",
+              "current_avg_millis": 303
+            },
+            "elapsed_time_tracker": {
+              "cancellation_count": 310,
+              "current_max": "1.3s",
+              "current_max_millis": 1305,
+              "current_avg": "649.3ms",
+              "current_avg_millis": 649
+            }
+          },
+          "cancellation_stats": {
+            "cancellation_count": 318,
+            "cancellation_limit_reached_count": 97
+          }
+        },
+        "mode": "enforced"
+      }
+    }
+  }
+}
+```
+
+### Response fields
+
+The response contains the following fields.
+
+Field Name | Data Type | Description
+:--- | :--- | :---
+search_backpressure | Object | Contains statistics about search backpressure.
+search_backpressure.<br>&nbsp;&nbsp;&nbsp;&nbsp;search_shard_task | Object | Contains statistics specific to the search shard task.
+search_backpressure.<br>&nbsp;&nbsp;&nbsp;&nbsp;search_shard_task.<br>&nbsp;&nbsp;&nbsp;&nbsp;[resource_tracker_stats](#resource_tracker_stats) | Object | Statistics about the current tasks.
+search_backpressure.<br>&nbsp;&nbsp;&nbsp;&nbsp;search_shard_task.<br>&nbsp;&nbsp;&nbsp;&nbsp;[calcellation_stats](#cancellation_stats) | Object |  Statistics about the canceled tasks since the node last restarted.
+search_backpressure.mode | String | The [mode](#search-backpressure-modes) for search backpressure. 
+
+### `resource_tracker_stats`
+
+The `resource_tracker_stats` object contains the statistics for each resource tracker: [`elapsed_time_tracker`](#elapsed_time_tracker), [`heap_usage_tracker`](#heap_usage_tracker), and [`cpu_usage_tracker`](#cpu_usage_tracker). 
+
+#### `elapsed_time_tracker`
+
+The `elapsed_time_tracker` object contains the following statistics related to the elapsed time.
+
+Field Name | Data Type | Description
+:--- | :--- | :---
+cancellation_count | Integer | The number of tasks canceled because of excessive elapsed time since the node last restarted.
+current_max_millis | Integer | The maximum elapsed time for all tasks currently running on the node, in milliseconds.
+current_avg_millis | Integer | The average elapsed time for all tasks currently running on the node, in milliseconds.
+
+#### `heap_usage_tracker`
+
+The `heap_usage_tracker` object contains the following statistics related to the heap usage.
+
+Field Name | Data Type | Description
+:--- | :--- | :---
+cancellation_count | Integer | The number of tasks canceled because of excessive heap usage since the node last restarted.
+current_max_bytes | Integer | The maximum heap usage for all tasks currently running on the node, in bytes.
+current_avg_bytes | Integer | The average heap usage for all tasks currently running on the node, in bytes.
+rolling_avg_bytes | Integer | The rolling average heap usage for the 100 most recent tasks, in bytes.
```

Comment:
nit: Rolling average is not hard-coded to work with just `100` most recent tasks.

This is a configurable setting defined by `search_backpressure.search_shard_task.heap_moving_average_window_size`. 100 is just the default value for it.